.github/workflows/ui-module-descriptor-verify.yml

```diff
@@ -0,0 +1,69 @@
+name: FOLIO UI Okapi Dependency Check
+
+on:
+  workflow_call:
+    inputs:
+      okapi-pull-contents:
+        description: JSON array of module descriptors to pull
+        required: true
+        type: string
+    secrets:
+      docker-username:
+        description: Username for pulling Okapi docker image
+        required: true
+      docker-password:
+        description: Password for pulling Okapi docker image
+        required: true
+
+jobs:
+  run:
+    name: Okapi dependency check
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker registry
+        run: echo "${{ secrets.docker-password }}" | docker login https://docker.io/v2/ -u "${{ secrets.docker-username }}" --password-stdin
```

Comment:
Any value in pulling config-y values like `https://docker.io/v2` into constants at the top of this file for visibility?

Reply:
eh? I don't think this would ever change unless we left dockerhub for some reason